packages/react-components/react-popover/stories/Popover/PopoverControllingOpenAndClose.stories.tsx

```diff
@@ -37,11 +52,7 @@ export const ControllingOpenAndClose = () => {
           <ExampleContent />
         </PopoverSurface>
       </Popover>
-
-      <label style={{ display: 'flex', gap: '10px', alignItems: 'center', marginTop: '10px' }}>
-        open
-        <input type="checkbox" name="state" value="open" checked={open} onChange={onChange} />
-      </label>
+      <Checkbox value="open" name="state" label="open" checked={open} onChange={onChange} className={styles.label} />
```

Comment:
I approve of the Fluent `Checkbox` usage  - this component was created before it existed 😅

However you should also modify the onChange function

```ts
 const onChange: CheckboxProps['onChange'] = (e, { checked }) => {
    setOpen(checked as boolean);
  };
```